### PR TITLE
Updating files to release

### PR DESCRIPTION
**Pull request by eddynaka**
## Changes:
- changed from pdbonly to portable
- removed the beta prerelease section
- added breaking change of default output sarif version

### CODE REVIEW

File: src/BinSkim.Sdk/BinaryAnalyzerContext.cs [hunk 1]
```diff
@@ -74,6 +74,7 @@ public string MimeType
         }
 
         public RuntimeConditions RuntimeErrors { get; set; }
+        public bool AnalysisComplete { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```
Comment (eddynaka) [inline src/BinSkim.Sdk/BinaryAnalyzerContext.cs:77]: Should I change to `{get;set;}`?
How should I use this?

Comment (michaelcfanning) [reply src/BinSkim.Sdk/BinaryAnalyzerContext.cs:77]: Yes, change to { get; set; } as you say. :)

---
In reply to: [521325596](https://github.com/microsoft/binskim/pull/300#discussion_r521325596) [](ancestors = 521325596)

File: src/Test.FunctionalTests.BinSkim.Driver/BaselineTests.cs [hunk 2]
```diff
@@ -1,228 +1,230 @@
...
+
+            var command = new AnalyzeCommand();
+            var options = new AnalyzeOptions
+            {
+                Force = true,
+                Verbose = true,
+                Recurse = false,
+                PrettyPrint = true,
+#pragma warning disable CS0618 // Type or member is obsolete
+                ComputeFileHashes = true,
```
Comment (michaelcfanning) [inline src/Test.FunctionalTests.BinSkim.Driver/BaselineTests.cs:106]: >ComputeFileHashes [](start = 16, length = 17)

Change this to:

DataToInsert = OptionallEmittedData.Hashes

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**